final_html_output.py

```diff
@@ -11,14 +11,26 @@ def final_html_output(df: pd.DataFrame) -> str:
         html string
     """
     html_string = """
-    <html>
-        <head><title>Opensource Leaderboard</title></head>
-        <link rel="stylesheet" type="text/css" href="style.css"/>
-        <body class="container">
-            <h1>OPENSOURCE LEADERBOARD</h1>
-            {table}
-        </body>
-    </html>
+  <html>
+    <head>
+      <title>Opensource Leaderboard Leapforg</title>
+    </head>
+    <link rel="stylesheet" type="text/css" href="../style.css" />
```

Comment:
We can put the stylesheets inside the `assets` directory.

Reply:
Ok sure